docs/contributing/coding-style.md

```diff
@@ -603,3 +629,27 @@ We do not suggest to modify the generated code other than the `tests` variable,
         }
         // generated test code
     ```
+
+1. Unused Variables
```

Comment:
sorry for misunderstandable comment before but I meant in this part we should write to delete the unused fields here